### ISSUE TITLE
Fix wifi autoconnect

### ISSUE THREAD

**Issue by rmburg** [HULKs/hulk#435]
If the selected Wi-Fi network is unreachable when the framework starts, the robot does not automatically connect to it once it becomes available.

**Comment by rkost** [comment]
The core of the issue seems to be that the nao currently does not scan for networks periodically (expect once at boot or something like this).

## Steps to reproduce

1. Boot nao with `SPL_B` **not** available. Wait for nao to boot completely
2. Make network `SPL_B` available
3. Execute pregame or wireless script: `./pepsi wireless set SPL_B 23`
4. Pepsi will fail with the following output (also after waiting some minutes):

```
./pepsi wireless set SPL_B 23
[10.1.24.23] ⣠ Setting network... Invalid network name 'SPL_B'
[10.1.24.23] ✗
   0: failed to set network on 10.1.24.23
   1: iwctl ssh command exited with exit status: 1
```

furthermore

```sh
./pepsi shell 23
iwctl station wlan0 get-networks
# shows available network list, no SPL_B
iwctl station wlan0 scan
# [WAIT some seconds]
iwctl station wlan0 get-networks
# Shows SPL_B as available network
exit
./pepsi wireless set SPL_B 23
# ok
```

TL;DR: Auto/periodic scan for wifi should fix this issue. Alternatively scan for networks before connecting to a new one.

**Comment by schmidma** [comment]
we don't use iwctl on the booster robot
